rundeckapp/grails-app/services/rundeck/services/ScheduledExecutionService.groovy

```diff
@@ -1518,6 +1518,7 @@ class ScheduledExecutionService implements ApplicationContextAware, Initializing
                 return [success: false, scheduledExecution: scheduledExecution,
                         errorCode: 'api.error.job.toggleSchedule.notScheduled' ]
             }
+            scheduledExecution.serverNodeUUID = frameworkService.isClusterModeEnabled()?frameworkService.serverUUID:null
```

Comment:
this change should also be tested if we expect the "update execution flags" to change the schedule ownership.  

e.g. If it was owned by another node, enabling the schedule would now reschedule the job on the current node, correct? I think that is the correct behavior, but it should be tested.